src/components/LanguagePicker/useLanguagePicker.tsx

```diff
@@ -123,8 +124,8 @@ export const useLanguagePicker = (
       return {
         localeOption,
         approvalProgress,
-        sourceName,
-        targetName,
+        sourceName: sourceName ?? "",
+        targetName: targetName ?? "",
```

Comment:
@pettinarip Seeing your comment [here](https://github.com/ethereum/ethereum-org-website/pull/12204#issuecomment-1948963646)... I went with this approach to avoid using mutable variables, sticking with `const`... lemme know if you disagree

Reply:
Wondering if we should fallback to default locale or something else. Not sure if `"'` isn't going to break something else in the chain tbh. If not, it ok.

Reply:
It wouldn't break anything, but it would just render without a name... I switched this to fallback to its language code.